### DISCUSSION TITLE
Checking a peptide

### DISCUSSION THREAD

**Discussion by jessicarose1974** [community/community#43820] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there, I am an R user. Do you have equivalent code for R?

**Comment by genidma** [comment]
What do you mean? Also, have you tried chat gpt? https://chat.openai.com/ You can ask chat gpt if it can write sample code for 'r package for peptides' for you. Thx

**Comment by jessicarose1974** [comment]
sorry i have never tried asking a question in the github environment. i was looking for help from Daniel Marrama on his https://github.com/IEDB/PEPMatch PEPmatch code.

Jess

"Sometimes in the morning, when it's a good surf, I go out there, and I don't feel like it's a bad world." Kary Mullis

Jessica Rose, PhD, MSc., BSc.
https://www.jessicasuniverse.com
https://jessica5b3.substack.com

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Reply by genidma** [reply]
Hi Doctor Rose. 

- I am also new to the community feature here. But I am 90% certain that your question is in the general section of Github.
- There are two contributors for the repository (repo) that you are interested in. 
- I took a look at the repo that you have shared. I personally did not see any mention regarding code for R in either the README.md file or the docs folder. 

You should try reaching out to the contributors to that repo directly. 
- I googled the name of one of the contributors. You may want to message them on Linkedin, [Twitter,](https://mobile.twitter.com/danielmarrama) ig. I don't see an option to message them on here. You could also contact the [institute ](https://www.lji.org/)directly and ask to be connected to the contact.
- Alternatively, you can search how to convert python code to R. But (and sample size is from me and very limited and I am not educated in this space). But it doesn't seem like there is an easy way to automatically convert python code to R. Mention of 'interfacing' but I am not sure what that means. [link](https://stackoverflow.com/questions/7933400/convert-python-to-r)

Kary Mullis invented the PCR procedure if memory serves me correctly. It's a procedure for replicating genetic information? And useful for a variety of tests today.
